biometricks/src/main/java/com/handstandsam/biometricks/Biometricks.kt

```diff
@@ -160,5 +163,19 @@ sealed class Biometricks {
                 }
             }
         }
+
+        /**
+         * Checks if we can securely authenticate, i.e. we have secure biometrics hardware and the user is
+         * enrolled. [androidx.biometric.BiometricManager.canAuthenticate] is unusable for this for a couple
+         * of reasons:
+         * 1. On api 28 it falls back to [FingerprintManager.hasEnrolledFingerprints] as the system method
+         * does not exist. However, there may still be non-fingerprint biometrics on those devices so it'll
+         * return a false-negative.
+         * 2. On api 29+ it checks for any form of biometrics, not just ones that are 'secure', so we can
+         * get a false-positive if the user is enrolled in an insecure form of biometrics.
+         */
+        fun canSecurelyAuthenticate(context: Context): Boolean {
```

Comment:
Looks like there aren't any tests in this repo, maybe this could be the PR that adds the first tests (to test this change)? 😬

Reply:
Yesss, we definitely need tests before shipping any alphas.  Kinda thing that we need to sit down and just do it but I am not finding that much time recently.  Maybe I can find some somewhere... At least finally responding to this!